Update adding-view.md (#35049)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/first-mvc-app/adding-view.md

Right-click on the *Views/HelloWorld* folder, and then **Add > New Item**.

> In the **Add New Item** dialog select **Show All Templates**. Visual Studio should by default show all templates without having to select **Show All Templates**.

Reviewer: ```suggestion
In the **Add New Item** dialog, if the option to **Show All Templates** is available, select it.
```
